run.go

```diff
@@ -207,7 +209,7 @@ func (s *step) echo(current, max int) {
 
 func (s *step) execute() error {
 	joinedCommand := strings.Join(s.command, " ")
-	cmd := exec.Command(bash, "-c", joinedCommand)
+	cmd := exec.Command(s.r.options.Shell, "-c", joinedCommand)
```

Comment:
Hm, we assume that every shell has a `-c` command, but I think that's fine for now.

Reply:
I believe this is a common thing - I've checked `sh`, `bash`, `ksh`, `zsh` and `fish` and they all use it